.deployment.config.json

```diff
@@ -191,16 +191,6 @@
         }
       }
     },
-    {
-      "id": "stg-release",
-      "team_jenkins": {
-        "disabled": true,
-        "job_name": "ui-kit-stg-release",
-        "stg": {
-          "disabled": false
-        }
-      }
-    },
     {
```

Comment:
Why is this step being removed?

Reply:
No longer needed. This executed the staging Jenkinsfile, which only updated the beta tag. Now, we already publish with the beta tag.